{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "19a81339_39138460",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 14
      },
      "lineNbr": 0,
      "author": {
        "id": 24824
      },
      "writtenOn": "2022-08-29T12:48:01Z",
      "side": 1,
      "message": "2022-08-27 06:30:23.908380 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 DEBUG unit.ovn-dedicated-chassis/1.install logger.go:60   Installing build dependencies: finished with status \u0027error\u0027\n2022-08-27 06:30:23.908477 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60   error: subprocess-exited-with-error\n2022-08-27 06:30:23.908551 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60\n2022-08-27 06:30:23.908626 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60   Ã— pip subprocess to install build dependencies did not run successfully.\n2022-08-27 06:30:23.908698 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60   â”‚ exit code: 1\n2022-08-27 06:30:23.908805 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60   â•°â”€\u003e [3 lines of output]\n2022-08-27 06:30:23.908884 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60       Looking in links: wheelhouse\n2022-08-27 06:30:23.908954 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60       ERROR: Could not find a version that satisfies the requirement setuptools\u003e\u003d42 (from versions: 41.6.0)\n2022-08-27 06:30:23.909024 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60       ERROR: No matching distribution found for setuptools\u003e\u003d42\n2022-08-27 06:30:23.909094 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60       [end of output]\n2022-08-27 06:30:23.909163 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60\n2022-08-27 06:30:23.909232 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60   note: This error originates from a subprocess, and is likely not a problem with pip.\n2022-08-27 06:30:23.909301 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60 error: subprocess-exited-with-error\n2022-08-27 06:30:23.909405 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60\n2022-08-27 06:30:23.909533 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60 Ã— pip subprocess to install build dependencies did not run successfully.\n2022-08-27 06:30:23.909649 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60 â”‚ exit code: 1\n2022-08-27 06:30:23.909767 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60 â•°â”€\u003e See above for output.\n2022-08-27 06:30:23.909900 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60\n2022-08-27 06:30:23.910060 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60 note: This error originates from a subprocess, and is likely not a problem with pip.\n2022-08-27 06:30:23.910174 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60 Traceback (most recent call last):\n2022-08-27 06:30:23.910282 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60   File \"/var/lib/juju/agents/unit-ovn-dedicated-chassis-1/charm/hooks/install\", line 8, in \u003cmodule\u003e\n2022-08-27 06:30:23.910389 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60     basic.bootstrap_charm_deps()\n2022-08-27 06:30:23.910503 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60   File \"/var/lib/juju/agents/unit-ovn-dedicated-chassis-1/charm/lib/charms/layer/basic.py\", line 224, in bootstrap_charm_deps\n2022-08-27 06:30:23.910640 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60     check_call([pip, \u0027install\u0027, \u0027-U\u0027, reinstall_flag, \u0027--no-index\u0027,\n2022-08-27 06:30:23.910754 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60   File \"/usr/lib/python3.10/subprocess.py\", line 369, in check_call\n2022-08-27 06:30:23.910864 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60     raise CalledProcessError(retcode, cmd)\n2022-08-27 06:30:23.910984 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 WARNING unit.ovn-dedicated-chassis/1.install logger.go:60 subprocess.CalledProcessError: Command \u0027[\u0027/var/lib/juju/agents/unit-ovn-dedicated-chassis-1/.venv/bin/pip\u0027, \u0027install\u0027, \u0027-U\u0027, \u0027--force-reinstall\u0027, \u0027--no-index\u0027, \u0027--no-cache-dir\u0027, \u0027-f\u0027, \u0027wheelhouse\u0027, \u0027Jinja2\u003d\u003d3.0.3\u0027, \u0027netaddr\u003d\u003d0.7.19\u0027, \u0027dnspython\u003d\u003d1.16.0\u0027, \u0027netifaces\u003d\u003d0.11.0\u0027, \u0027six\u003d\u003d1.16.0\u0027, \u0027pyrsistent\u003d\u003d0.18.0\u0027, \u0027zipp\u003d\u003d3.6.0\u0027, \u0027psutil\u003d\u003d5.9.1\u0027, \u0027importlib-metadata\u003d\u003d4.8.3\u0027, \u0027PyYAML\u003d\u003d5.3.1\u0027, \u0027pbr\u003d\u003d5.10.0\u0027, \u0027typing-extensions\u003d\u003d4.1.1\u0027, \u0027jsonschema\u003d\u003d3.2.0\u0027, \u0027charms.openstack\u003d\u003d0.0.1.dev1\u0027, \u0027attrs\u003d\u003d22.1.0\u0027, \u0027pyaml\u003d\u003d21.10.1\u0027, \u0027charms.reactive\u003d\u003d1.5.0\u0027, \u0027charmhelpers\u003d\u003d1.1.1.dev37\u0027]\u0027 returned non-zero exit status 1.\n2022-08-27 06:30:23.911101 | focal-medium | 2022-08-27 06:30:23 [ERROR] unit-ovn-dedicated-chassis-1.log: 2022-08-27 06:29:20 ERROR juju.worker.uniter.operation runhook.go:140 hook \"install\" (via explicit, bespoke hook script) failed: exit status 1\n",
      "revId": "63ecaf8bdb4e93785e5b9f911a5a96ba705c007e",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "e53a9286_4716a604",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 14
      },
      "lineNbr": 0,
      "author": {
        "id": 24824
      },
      "writtenOn": "2022-08-29T19:28:23Z",
      "side": 1,
      "message": "rechecking since https://github.com/openstack-charmers/charm-layer-ovn/pull/64 is merged",
      "revId": "63ecaf8bdb4e93785e5b9f911a5a96ba705c007e",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "2ffb956e_6e326406",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 14
      },
      "lineNbr": 0,
      "author": {
        "id": 24824
      },
      "writtenOn": "2022-08-29T19:28:28Z",
      "side": 1,
      "message": "recheck",
      "revId": "63ecaf8bdb4e93785e5b9f911a5a96ba705c007e",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543"
    }
  ]
}